### PR TITLE
Add banksy-testnet-3

### PR DESCRIPTION
**Pull request by vuong177**
Add banksy-testnet-3 initial RS testnet registration.

### CODE REVIEW

File: replicated-security/banksy-testnet-3/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+- Public Notional endpoints: 
+    - RPC: `https://rpc-banksy.notional.ventures:443`
+    - API: `https://api-banksy.notional.ventures:443`
+    - gRPC: `https://grpc-banksy.notional.ventures:443`
+- Block Explorer: `https://explorer.nodexcapital.com/banksy-testnet`
+
+### IBC data
+Available soon
+
+### Joining berfore transition
```
Comment (dasanchez) [inline replicated-security/banksy-testnet-3/README.md:36]:
```suggestion
### Joining before transition
```

File: replicated-security/banksy-testnet-3/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+```bash
+# Installing centaurid v6.0.0-ics
+export PATH=$PATH:$HOME/go/bin
+cd $HOME
+git clone https://github.com/notional-labs/composable-centauri
+cd composable-centauri
+git checkout v6.0.0-ics # Using v6.0.0-ics
+make install
+centaurid version # v6.0.0-ics
```
Comment (dasanchez) [inline replicated-security/banksy-testnet-3/README.md:79]: This returns `v5.1.0`, can you double-check please?

**Review by dasanchez — COMMENTED** [review]
Thanks @vuong177 !

File: replicated-security/banksy-testnet-3/README.md (hunk 2 above)
Comment (MasterPi-2124) [reply replicated-security/banksy-testnet-3/README.md:79]: yes it is a problem in our code. Our team are fixing that

**Pull request merged**